FrontEnd/src/main/java/resourceDisplay/AssetsRepository.java

```diff
@@ -10,6 +10,31 @@
  */
 public class AssetsRepository {
 
+    /**
+     * This is used to create a list of all workspace values for a specific user
+     *
+     * @param dbClient    - A client for connection to the DB
+     * @param userID - The user for whom we want the list of workspaces
+     * @return a list of all workspace values for the specific user
+     */
+    public List<String> getWorkspaceIdList(DatabaseClient dbClient, String userID) {
+        List<String> workspaceIdList = new ArrayList<>();
+        String statementString = "SELECT DISTINCT workspaceId FROM Workspace_User_Table WHERE " +
+                "userEmail = @userID ORDER BY workspaceId";
```

Comment:
this string is affected by input, could be created as a static member so it won't be recreated each time we call this method

Reply:
You want me to save the result to a static member and when the method is called see if we already have it and not query for it again?

Reply:
not the result, just the `statementString` var - it doesn't seem like it changes in each run so we can save it statically.